### PR TITLE
NP-1672 Add handling for Content-Range header

### PR DESCRIPTION
**Pull request by brinxmat**
Please refer to RFC 7231 whenever you define a new API so that we avoid having to make changes like this in future.

If a Content-Range header is present on a PUT request, according to RFC 7231 §4.3.4, the service should return 400 BAD REQUEST as the request indicates that the request contains partial content.

### CODE REVIEW

File: modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java [hunk 1]
```diff
@@ -20,6 +21,7 @@
 import java.util.UUID;
 
 import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.apache.http.HttpHeaders.CONTENT_RANGE;
 
 public class ModifyPublicationHandler extends ApiGatewayHandler<UpdatePublicationRequest, PublicationResponse> {
```
Comment (norrs) [inline modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java:26]: Could maybe ApiGatewayHandler itself do the verifying?

default method and ensure we call super?

like: if method.PUT && !validateNotPartialContentReuqest  ; bad boy.

**Review by norrs — CHANGES_REQUESTED** [review]
I guess we might end up with adding more default validations later if API gateway from Amazon lets em thru. We shouldnt need to remember to check this manually for each handler we have.

File: modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java (hunk 1 above)
Comment (axthosarouris) [reply modify-publication/src/main/java/no/unit/nva/publication/modify/ModifyPublicationHandler.java:26]: Maybe we should have have a PutRequestHander and a (GetRequestHandler etc.) where each one of them checks for valid requests in their context (PUT,GET,POST, etc) requests. So we can void the clutter of 
```
if(method.name==something){
 validate()
}

**Review by axthosarouris — APPROVED** [review]
Approved under the condition that we create a task for validating requests.

**Comment by norrs** [comment]
@brinxmat Link to tech debt if not fixing it right away.

**Review by janbj — APPROVED** [review]

**Pull request closed without merging**